homeassistant/components/overkiz/const.py

```diff
@@ -94,3 +114,42 @@
     OverkizCommandParam.SFC: "sfc",
     OverkizCommandParam.UPS: "ups",
 }
+
+OVERKIZ_UNIT_TO_HA: dict[str, str] = {
+    MeasuredValueType.ABSOLUTE_VALUE: "",
+    MeasuredValueType.ANGLE_IN_DEGREES: DEGREE,
+    # MeasuredValueType.ANGULAR_SPEED_IN_DEGREES_PER_SECOND: "",
```

Comment:
Do you plan tu use our own constants (or just a string, even easier) for the units that are not provided by HA?

Reply:
That’s something we are allowed to do? If yes, I can do indeed.

Reply:
Yes (if I am not mistaken), we can create them if we want.

Reply:
Fixed.